.makefile

```diff
@@ -0,0 +1,127 @@
+# BUILD_LATEST   # Optional.  Will build 'latest' components if defined.  Default is off.
+#
+
+ifndef CHART
+$(error CHART is not set)
+endif
+
+ifndef RELEASE_NAME
+$(error RELEASE_NAME is not set)
+endif
+
+# Default value for namespace used with helm.  Override if required.
+NAMESPACE ?= lightbend
+
+# Will build 'latest' bits if this is defined.  Override if desired.
+BUILD_LATEST ?=
+
+## These variables usable inside the Makefiles that include this one.
+#
+
+COMPONENTS := $(wildcard */.)
+SUBCOMPONENTS := $(wildcard */*/.)
+
+HELM_CHARTS_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+SCRIPTS_DIR := $(HELM_CHARTS_DIR)/scripts
+
+define banner
+	$(info === $@)
+endef
+
+
+all: init test build
+build:: init $(HELM_CHARTS_DIR)/docs/index.yaml $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/all.yaml
+
+export VERSION := $(shell $(SCRIPTS_DIR)/export-chart-version.sh $(CHART))
+export RELEASE := $(CHART)-$(VERSION)
+
+$(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/all.yaml: $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz
+	$(call banner)
+	mkdir -p $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)
+	helm --namespace=$(NAMESPACE) template $< > $@
+
+$(HELM_CHARTS_DIR)/docs/index.yaml: $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz #docs/$(RELEASE_LATEST).tgz
+	$(call banner)
+	helm repo index $(HELM_CHARTS_DIR)/docs --url https://lightbend.github.io/helm-charts
+
+$(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz: $(COMPONENTS) $(SUBCOMPONENTS)
+	$(call banner)
+	helm package -d $(HELM_CHARTS_DIR)/docs .
+
+
+## Conditionally add rules etc. for building 'latest' images.  Not done by default.
+##
+ifdef BUILD_LATEST
+CHART_LATEST := $(CHART)-latest
+export RELEASE_LATEST := $(CHART_LATEST)-$(VERSION)
+
+build:: $(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/all-latest.yaml
+
+$(HELM_CHARTS_DIR)/docs/$(RELEASE_NAME)/all-latest.yaml: $(HELM_CHARTS_DIR)/docs/$(RELEASE_LATEST).tgz
+	$(call banner)
+	helm --namespace=$(NAMESPACE) template $< > $@
+
+$(HELM_CHARTS_DIR)/docs/$(RELEASE_LATEST).tgz: $(COMPONENTS) $(SUBCOMPONENTS)
+	$(call banner)
+	rm -rf $(HELM_CHARTS_DIR)/build/$(CHART_LATEST)
+	cp -r . $(HELM_CHARTS_DIR)/build/$(CHART_LATEST)
+	$(SCRIPTS_DIR)/munge-to-latest.sh $(HELM_CHARTS_DIR)/build/$(CHART_LATEST)
+	helm package -d $(HELM_CHARTS_DIR)/docs $(HELM_CHARTS_DIR)/build/$(CHART_LATEST)
+
+install-local-latest: $(HELM_CHARTS_DIR)/docs/$(RELEASE_LATEST).tgz install-helm delete-local
+	helm install $(HELM_CHARTS_DIR)/docs/$(RELEASE_LATEST).tgz --name=$(RELEASE_NAME) --namespace=$(NAMESPACE) --debug --wait
+
+.PHONY: install-local-latest
+
+endif
+##
+## end conditional things for 'latest' builds
+
+
+# Not sure what this is/was about...
+# # duplicate method here to generate the index, to avoid pulling in RELEASE as a dependency
+# latest: init test docs/$(RELEASE_NAME)/all-latest.yaml docs/$(RELEASE_LATEST).tgz
+# 	helm repo index docs --url https://lightbend.github.io/helm-charts
+
+# What is this?
+release:
+	$(call banner)
+	echo "stub: release"
```

Comment:
I think I added this as part of the travis automation because I wanted it to do a release on merge to master. But I think I ripped that out.